commands/FBDebugCommands.py

```diff
@@ -189,6 +191,86 @@ def run(self, arguments, options):
     fb.evaluateEffect('[[UIApplication sharedApplication] performSelector:@selector(_performMemoryWarning)]')
 
 
+def switchBreakpointState(expression,on):
+  ci = lldb.debugger.GetCommandInterpreter()
+  res = lldb.SBCommandReturnObject()
+
+  expression_pattern = re.compile(r'{}'.format(expression),re.I)
+
+  target = lldb.debugger.GetSelectedTarget()
+  for breakpoint in target.breakpoint_iter():
+    for location in breakpoint:
+      if expression_pattern.search('{}'.format(location)):
```

Comment:
This also doesn't need the `format`, I think it can be just `.search(str(location))`.